NSS-Day-Cohort-42/rare-media-moguls
# Category Form

#### Changes Made
1. Added `CategoryForm.js` to categories folder with form to add new category.
2. Added Category Management link to NavBar.js
3.  Added `CategoriesList.js` to categories folder with button to Add New Category.

#### Steps to Review
1. Checkout this branch locally and run the application.
```
git fetch --all
git checkout sm-create-category
```
2. Run the application.
```
npm start
```
3. Open a new Terminal tab (⌘T) and navigate to the server directory.
```
..
server
```
4. Initialize virtual environment, and run the server.
```
pipenv shell
watchgod request_handler.main
```
5. Test app functionality. 
>Open dev tools in the application and select Application from the menu at the top.
>Open Local Storage in the left sidebar of dev tools, and select the URL shown.
>Enter rare_user_id as the Key and 1 as the Value, then refresh the page.
>You should be redirected to the home page as a logged in user.
>Click on Category Management link in Nav Bar. See if Button renders on page. 
>Click button to check that form renders to page. Enter information. 
> Once save button is clicked, user should be taken back to categories list page. (nothing will render)


6. View code file.
> Confirm file modifications are present as indicated above.
> Confirm no unused code or extraneous comments exist.
